packages/main/src/themes/ColorPalette.css

```diff
@@ -21,10 +22,44 @@
 }
 
 .ui5-cp-root.ui5-cp-root-phone .ui5-cp-item-container {
-	width: 18.5rem;
-	max-width: 19.5rem;
-	max-height: 13rem;
 	padding: 0.375rem 0.625rem;
+	width: 95%;
+	max-width: 95%;
+	max-height: 100%;
+}
+
+.ui5-cp-root-phone .ui5-cp-default-color-button,
+.ui5-cp-root-phone .ui5-cp-more-colors {
+	color: var(--sapTextColor);
+	height: 2.875rem;
+}
+
+.ui5-cp-root-phone .ui5-cp-more-colors-wrapper,
+.ui5-cp-root-phone .ui5-cp-default-color-button-wrapper {
+	height: 2.875rem;
+}
+
+.ui5-cp-root-phone .ui5-cp-item-container{
+	display: grid;
+    grid-template-columns: 18% 18% 18% 18% 18%;
+    grid-gap: 0.375rem;
+}
+
+.ui5-cp-root-phone .ui5-cp-recent-colors-container {
+	display: grid;
+    grid-template-columns: 18% 18% 18% 18% 18%;
```

Comment:
I think using percentages is not a good practice as it often results in not whole numbers as values after calculated. This may cause inconsistencies on different mobile devices

Reply:
Using percentage is neither good or bad practice. It's as good as using px or rem. It's situational use.  Based on our specifications we don't really have a choice but using vh/vw or pecentages. Yes, media query is an option but we will need 10 of them just to fit most of the cases.